core/migrations/0008_instance_actions_and_migrate_provider_trait_to_dns_and_auto_imaging.py

```diff
@@ -100,6 +100,9 @@ def add_instance_actions(Provider, InstanceAction, ProviderInstanceAction):
         name="Reboot",
         description="""Reboots an instance when it is in ANY State""")
     InstanceAction.objects.get_or_create(
+        name="Redeploy",
+        description="""Redeploy to an instance when it is in ANY active state""")
+    InstanceAction.objects.get_or_create(
```

Comment:
I'll go ahead and create a new migration for clarity (rather than update) -- however any provider that was created *after running the migrations* would not have new (any) ProviderInstanceActions.. To remedy this I included `def verify_actions_exist(cls, all_actions):` in `core/models/instance_action.py#72` which will ensure all available actions (based on logic in `valid_instance_actions` are available for each provider).